lib/app_server.py

```diff
@@ -219,9 +219,19 @@ def kill():
         self.process.wait()
         timer.cancel()
 
+    @classmethod
+    def version(cls):
+        p = subprocess.Popen([cls.binary, "--version"], stdout=subprocess.PIPE)
+        version = bytes_to_str(p.stdout.read()).rstrip()
+        p.wait()
+        return version
```

Comment:
Can we reuse existing `version()` classmethod from TarantoolServer in some way (directly or by moving it to Server or even to utils)?

Reply:
Moved to Server class